server/upgradeschema/upgrade_pre31.go

```diff
@@ -0,0 +1,126 @@
+		// InstanceStatusChangeType is the StatusUpdate type for an instance state change event
+		InstanceStatusChangeType StatusUpdateType = iota
+		// DeploymentStatusChangeType is the StatusUpdate type for an deployment status change event
+		DeploymentStatusChangeType
+		// CustomCommandStatusChangeType is the StatusUpdate type for an custom command status change event
+		CustomCommandStatusChangeType
+		// ScalingStatusChangeType is the StatusUpdate type for an scaling status change event
+		ScalingStatusChangeType
+		// WorkflowStatusChangeType is the StatusUpdate type for an workflow status change event
+		WorkflowStatusChangeType
+	)
+
+	stringStatusUpdate := func(s StatusUpdateType) string {
+		switch s {
+		case InstanceStatusChangeType:
+			return events.StatusChangeTypeInstance.String()
+		case DeploymentStatusChangeType:
+			return events.StatusChangeTypeDeployment.String()
+		case CustomCommandStatusChangeType:
+			return events.StatusChangeTypeCustomCommand.String()
+		case ScalingStatusChangeType:
+			return events.StatusChangeTypeScaling.String()
+		case WorkflowStatusChangeType:
+			return events.StatusChangeTypeWorkflow.String()
+		}
+		return ""
+	}
+
+	kvps, qm, err := kv.List(eventsPrefix, nil)
+	if err != nil || qm == nil {
+		return errors.Wrapf(err, "failed to upgrade consul database schema from 100")
+	}
+	for _, kvp := range kvps {
+		event := make(map[string]string)
+		depIDAndTimestamp := strings.Split(strings.TrimPrefix(kvp.Key, eventsPrefix+"/"), "/")
+		deploymentID := depIDAndTimestamp[0]
+		eventTimestamp := depIDAndTimestamp[1]
+
+		values := strings.Split(string(kvp.Value), "\n")
+		eventType := StatusUpdateType(kvp.Flags)
+		switch eventType {
+		case InstanceStatusChangeType:
+			if len(values) != 3 {
+				return errors.Errorf("failed to upgrade consul database schema from 100: unexpected event value %q for event %q", string(kvp.Value), kvp.Key)
+			}
+			event["deploymentId"] = deploymentID
+			event["timestamp"] = eventTimestamp
+			event["type"] = stringStatusUpdate(eventType)
+			event["nodeId"] = values[0]
+			event["status"] = values[1]
+			event["instanceId"] = values[2]
+		case DeploymentStatusChangeType:
+			if len(values) != 1 {
+				return errors.Errorf("failed to upgrade consul database schema from 100: unexpected event value %q for event %q", string(kvp.Value), kvp.Key)
+			}
+			event["deploymentId"] = deploymentID
+			event["timestamp"] = eventTimestamp
+			event["type"] = stringStatusUpdate(eventType)
+			event["status"] = values[0]
+		case CustomCommandStatusChangeType, ScalingStatusChangeType, WorkflowStatusChangeType:
+			if len(values) != 2 {
+				return errors.Errorf("failed to upgrade consul database schema from 100: unexpected event value %q for event %q", string(kvp.Value), kvp.Key)
+			}
+			event["deploymentId"] = deploymentID
+			event["timestamp"] = eventTimestamp
+			event["type"] = stringStatusUpdate(eventType)
+			event["status"] = values[1]
+			event["alienExecutionId"] = values[0]
```

Comment:
Shouldn't it be : event["alienTaskId"] = values[0] ?
(alienTaskId and not alienExecutionId)

Reply:
Alien Task is a Yorc execution.
Alien execution is a Yorc task.
Maybe we should rename these concepts in a future major release.

Reply:
ok, would just add a comment to warn the reader about this naming issue between Yorc and Alien4Cloud